doc/gov/chapters/chapter09.md

```diff
@@ -81,3 +81,321 @@ Exceptions will be clearly recorded in Reference Architectures' Appendices which
+
+<a name="9.5.2.7.2"></a>
+#### 9.5.2.7.2 Trial Participant Interaction with the Community
+
+The focus of the field trials is on the test suites and CNTT methods, not on the systems under test. A process is being developed to identify issues and gaps and managing how they are reported.
+
+CNTT will work very closely with field trial partners (NFVI vendors, VNF vendors, or system integrators) and agree on labs that will be used for the trial. CNTT will take all necessary measures to protect the intellectual property rights (IP rights) for all partners involved in those trials. All Reports and findings will be vetted carefully and only published after being approved by all parties concerned. No test results or records will be kept in any public records without the consent of the participants.
+
+The targeted repositories for this information are:
+
+CNTT GitHub
+
+*  GitHub Code
+  
+*  GitHub Projects
+  
+*  GitHub Issues
+  
+OPNFV 
+
+*  (Where?)
+
+<a name="9.5.2.7.3"></a>
+#### 9.5.2.7.3 Test Case Identification
+
+Specific test cases for the field trials will be documented and provided to the participants based upon the CNTT RI-1  and RC-1 work streams requirements. The technical testing methods, procedures and documentation shall be provided by these work streams. 
+
+<a name="9.5.2.7.4"></a>
+#### 9.5.2.7.4 Vendor Solicitation/Commitment
+
+NFVI and VNF vendor members will be solicited for participation in the trials. The vendors will be required to commit fully to the assessment and evaluation  processes. As previously mentioned, additional discussion is needed to define what results data and at what level is acceptable to be shared.
+
+<a name="9.5.2.7.5"></a>
+#### 9.5.2.7.5 Deliverable
+
+The Initiate Field Trial Stage will deliver execution and assessment plans including: 
+
+* A high-level check list of the tasks each participant will need to complete shall be provided.  
+* The plan will contain all the key milestones and activities the participants will expected to perform. 
+
+<a name="9.5.2.8"></a>
+### 9.5.2.8 Execution Stage
+
+<a name="9.5.2.8.1"></a>
+#### 9.5.2.8.1 Objectives of the Execute Stage
+
+The objective of Execute Stage is participants implementing field trials tasks and record/assess outcomes CNTT will assemble the Trials team to fully develop the action plan including resource assignments, materials requirements, and timelines. 
+
+Activities include the deployment and configuration of RI-1 and execution of the RC-1 test cases. Vendor community members that commit to the trials will build/setup/prep labs for the trials per the instructions:
+
+1.	Secure appropriate environment space (pre-existing, new build, LaaS)
+
+2.	RI-1 implementation  per published RI-1 Specifications  
+
+3.	RC-1 Test suite will be provided to the participants (MF for detail)
```

Comment:
it's already in CNTT RC.
https://cntt-n.github.io/CNTT/doc/ref_cert/lfn/chapters/chapter03.html#3.4

Reply:
Cedric, completely agree that it is there and very complete. It is evident that great care was taken to create it. The one area of concern is readability of the average System Admin that would be responsible for RI implementation and RC execution.  Others in the community should be engaged to help bring the test suite to a lowest common denominator level.

Reply:
@collivier is functest complete to test all of RA-1 requirements, for example, RBAC access roles/rights. If not, would the CNTT compliance evolve to include other tests beyond functest.